deployments/helm/nvidia-dra-driver-gpu/templates/_helpers.tpl

```diff
@@ -80,7 +80,7 @@ Selector labels
 Full image name with tag
 */}}
 {{- define "nvidia-dra-driver-gpu.fullimage" -}}
-{{- $tag := printf "%s" .Chart.AppVersion }}
+{{- $tag := printf "v%s" .Chart.AppVersion }}
```

Comment:
In the future we should add a code comment here, too, if the template language allows for that.